include/glow/Base/Tensor.h

```diff
@@ -524,7 +524,14 @@ template <class ElemTy> class Handle final {
     assert(sizes.size() > 1 && "Tensor has only one dimension");
     assert(idx < sizes[0] && "Invalid first index");
     auto elemTy = tensor_->getElementType();
-    Tensor slice(elemTy, sizes.slice(1));
+
+    Tensor slice;
+    auto &ty = tensor_->getType();
+    if (ty.isQuantizedType()) {
```

Comment:
consider using Type.newShape which encapsulates the complexity of handling quantized types.

Reply:
Thank you!